metricbeat/operations/default.json

```diff
@@ -0,0 +1,65 @@
+    {
+      "name": "index-append",
+      "operation-type": "bulk",
+      "bulk-size": {{bulk_size | default(10000)}},
+      "ingest-percentage": {{ingest_percentage | default(100)}}
+    },
+    {
+      "name": "default",
```

Comment:
Are you interested in measuring performance of `match_all`? If not I suggest to remove it from the track.